### DISCUSSION TITLE
Game tells you to be able to plant seed when planter box is dropped

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#15188] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

It is misleading which confuses newer players when it tells them they can plant while the planter box is dropped and not placed.

### Reproduction steps

1. Drop planter box
2. Equip a seed
3. Hover your cursor over planter box and observe the available actions

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this.
Tooltip now shows "pickup" when planter is not attached, even with seed in hand.
Closing discussion as resolved.
